site/docs/use/00-user.md

```diff
@@ -8,20 +8,127 @@ title: User Manual
 
 ### Self Register User
 
+Zitadel allows users to register a organization and/or user with just a few steps.
+
+1. Register a organization
+2. Verify your email
+3. Login to Zitadel and manage the organization
+
+
+<div class="zitadel-gallery" itemscope itemtype="http://schema.org/ImageGallery">
+    <figure itemprop="associatedMedia" itemscope itemtype="http://schema.org/ImageObject">
+        <a href="img/register.gif" itemprop="contentUrl" data-size="1100x906">
+            <img src="img/register.gif" itemprop="thumbnail" alt="Self Register" />
+        </a>
+        <figcaption itemprop="caption description">Self Register</figcaption>
+    </figure>
+</div>
+
+
 ### Verify EMail
```

Comment:
I would add this section to the register user section, because its finish initialization. And then add a new section Change E-Mail, because E-Mail Verify Mail is not the same as initialization Mail

Reply:
agree, changed it to "change email" and put the initial registration above